modules/ossm-automatic-sidecar-injection.adoc

```diff
@@ -6,20 +6,24 @@
 [id="ossm-automatic-sidecar-injection_{context}"]
 = Enabling automatic sidecar injection
 
-When deploying an application into the {ProductName} you must opt in to injection by specifying the `sidecar.istio.io/inject` annotation with a value of `"true"`. Opting in ensures that the sidecar injection does not interfere with other OpenShift features such as builder pods used by numerous frameworks within the OpenShift ecosystem.
+When deploying an application, you must opt-in to injection by setting the `sidecar.istio.io/inject` annotation to `"true"`. Opting in ensures that the sidecar injection does not interfere with other OpenShift features such as builder pods used by numerous frameworks within the OpenShift ecosystem. Automatic injection using annotation removes the need to label projects.
```

Comment:
Is "removes the need to label projects" appropriate here? 

From an OCP point of view -- labeling projects requires additional privileges that not all users have so in that sense "removes the need" is correct

From the point of view of someone who knows upstream Istio -- the user is likely familiar with labeling projects to have all pods in that project injected with the sidecar. In this case, we aren't removing the need for anything, but rather using a different behavior with the labeling of the project not being supported.